doc/plugin_server_nodeattestor_aws_iid.md

```diff
@@ -45,6 +48,27 @@ In the following configuration,
 
 assuming AWS IID document sent from the spire agent contains `accountId : 12345678`, the spire server will assume "arn:aws:iam::12345678:role/spire-server-delegate" role before making any AWS call for the node attestation. If `assume_role` is configured, the spire server will always assume the role even if the both the spire-server and the spire agent is deployed in the same account.
 
+##### Enabling AWS Node Attestation Organization Validation
```

Comment:
The config part of this section can move into a new table as I mentioned above. We can keep some of the language about what IAM policies are required in order for us to do this. Sample IAM policy would be very helpful here.